app/_hub/kong-inc/openid-connect/how-to/_fapi.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Financial-Grade API (FAPI)
+nav_title: Financial-Grade API (FAPI)
+minimum_version: 3.7.x
+---
+
+
+The OpenID Connect plugin supports various features of the FAPI standard, aimed to protect APIs that expose high-value and sensitive data.
+
+### Pushed Authorization Requests (PAR)
+
+With PAR enabled, Kong (OAuth client) sends the payload of an authorization request to the IdP. As a result, it obtains a `request_uri` value. This value is used by the client in a call to the authorization endpoint as a reference to obtain the authorization request payload data.
+
+For more information refer to the [Configuration reference](/hub/kong-inc/openid-connect/configuration/#config-pushed_authorization_request_endpoint)
+
+### JWT-Secured Authorization requests (JAR)
+
+With JAR enabled, when sending requests to the authorization endpoint, Kong provides request parameters in a JSON Web Token (JWT) instead of using a query string, which allows the request data to be signed with JSON Web Signature (JWS).
```

Comment:
```suggestion
With JAR enabled, when sending requests to the authorization endpoint, Kong provides request parameters in a JSON Web Token (JWT) instead of using a query string. This allows the request data to be signed with JSON Web Signature (JWS).
```
Splitting run-on sentence